app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+    before_save { self.email = email.downcase }
```

Comment:
```suggestion
    before_save {self.email = email.downcase}
```
với sau { hoặc } k có space nhá.